src/Plugin/Block/SearchResponseBlock.php

```
@@ -3,58 +3,57 @@
  *   id = "ai_search_block_response",
- *   admin_label = @Translation("AI Search response"),
+ *   admin_label = @Translation("AI Search Response"),
  *   category = @Translation("AI")
  * )
  */
 class SearchResponseBlock extends BlockBase implements ContainerFactoryPluginInterface {
 
-  /**
-   * {@inheritdoc}
-   */
   public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$container'. (Unused Code Rules)

(UnusedFormalParameter)